### ISSUE TITLE
[PM] Put together Profile 2.0 Release Plan

### ISSUE THREAD

**Issue by mshea0606** [department-of-veterans-affairs/va.gov-team#10354]
## Tasks
- [x] Fill out portions of the release plan that can be completed
- [x] Review with the team
- [x] Revise if neccessary

**Comment by mshea0606** [comment]
@Samara-Strauss Here is the [Profile 2.0 Release Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/Combine%20Profile%20and%20Account/Profile%202.0%20Release%20Plan.md)

**Comment by Samara-Strauss** [comment]
Thanks for this!

- Added some notes for the “Design/FE/BE UAT plan” sections. Idk exactly what they are looking for here but filled in what we know now.
- Under the “How will you make the product available in production while limiting the number of users who can access it?”, I extended the period of time at 50% to 1 full week. I don’t think there is a right or wrong amount of time to stay at 50% for every project. For this project, I think we should stay at 50% for multiple days just because it’s such a large effort and will be so new to folks. I want to give us enough time to make sure we’re not getting in complaints from the call center or seeing other issues before we launch to everyone.
- Under “What metrics-based criteria will you look at…”, I removed successful updates since I think that is the same as successful submissions. I’m pretty sure that means people who submitted whose informations saved successfully. Also removed save failures since I think that’s the same as failed form submissions.
